Source/santad/EventProviders/SNTEndpointSecurityDeviceManager.mm

```diff
@@ -211,6 +211,18 @@ - (instancetype)initWithESAPI:(std::shared_ptr<EndpointSecurityAPI>)esApi
   return self;
 }
 
+- (uint32_t)updatedMountFlags:(struct statfs*)sfs {
+  uint32_t mask = sfs->f_flags | mountArgsToMask(self.remountArgs);
+
+  // NB: APFS mounts get MNT_JOURNALED implicitly set. However, mount_apfs
+  // does not support the `-j` option so this flag needs to be cleared.
+  if (strncmp(sfs->f_fstypename, "apfs", sizeof(sfs->f_fstypename)) == 0) {
```

Comment:
nit: Couldn't this just be `(strncmp(sfs->f_fstypename, "apfs", 4) == 0)`?

Reply:
eh, six of one... Technically would want 5 to include the null terminator. Given `strncmp` will stop at the first null terminator of either string it doesn't really matter. Doing something like `strncmp(sfs->f_fstypename, "apfs", sizeof("apfs"))` just looks weird to me.